mandatory/3-function-output.js

```diff
@@ -5,12 +5,13 @@ function getRandomNumber() {
 
 // Add comments to explain what this function does. You're meant to use Google!
```

Comment:
Hi, I believe you are supposed to write your own comment here to describe what this function does.